pages/result.html

```diff
@@ -0,0 +1,23 @@
+<html>
+  <head>
+    <title>True Quiz</title>
```

Comment:
Not mandatory but you can use meta tags for responsive design, author, and description. Its a good practice